js/main.js

```diff
@@ -107,107 +130,69 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 // switch to show or hide no entry text
-const $noEntries = document.querySelector('.center');
+const $noEntries = document.querySelector('#center');
 function toggleNoEntries() {
-  if (data.entries.length > 0) {
-    $noEntries.setAttribute('class', 'hidden center');
+  if (data.entries.length < 1) {
+    $noEntries.className = 'center';
   } else {
-    $noEntries.setAttribute('class', 'center');
+    $noEntries.className = 'hidden';
   }
 }
 
 // viewswapping between entries and entry-form pages
 function viewSwap(viewName) {
   if (viewName === 'entry-form') {
-    $entries.setAttribute('class', 'hidden');
-    $entryForm.setAttribute('class', 'display');
+    $entries.classList.add('hidden');
+    $entryForm.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
-  } else {
-    $entries.setAttribute('class', 'display');
-    $entryForm.setAttribute('class', 'hidden');
+  } else if (viewName === 'entries') {
+    $entryForm.classList.add('hidden');
+    $entries.classList.remove('hidden');
     data.view = viewName;
-    toggleNoEntries();
   }
 }
 
 // shows Entries page when the Entries tab is clicked
 $nav.addEventListener('click', function () {
+  data.view = 'entries';
   viewSwap('entries');
 });
 
 // shows the Entry Form page when NEW button is clicked
 $newButton.addEventListener('click', function () {
+  $form.reset();
+  $imgSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $pageName.textContent = 'New Entries';
+  data.view = 'entry-form';
   viewSwap('entry-form');
 });
 
-// switch to Edit Entries page when pen icon is clicked
-$ul.addEventListener('click', function (event) {
-  const $li = event.target.closest('li'); // grabs the closest <li> element from the event (pen icon);
+// modal pop up when user clicks delete entry
+const $modal = document.querySelector('.modal');
+
+$delButton.addEventListener('click', function (event) {
+  $modal.classList.remove('hidden');
+});
+// modal cancel delete button and exits modal
+const $modalCancelBtn = document.querySelector('.cancel');
+$modalCancelBtn.addEventListener('click', function () {
+  $modal.classList.add('hidden');
+});
+
+// modal confirm delete button and deletes that form entryId from data storage
+const $modalConfirmBtn = document.querySelector('.confirm');
+
+$modalConfirmBtn.addEventListener('click', function () {
+  $modal.classList.add('hidden');
+  const $li = document.querySelectorAll('li');
   for (let i = 0; i < data.entries.length; i++) {
-    if (data.entries[i].entryId === Number($li.getAttribute('data-entry-id'))) {
-      data.editing = data.entries[i];
+    if (data.entries[i].entryId === Number(data.editing.entryId)) {
+      data.entries.splice([i], 1);
+      $ul.removeChild($li[i]);
+      data.editing = null;
+      data.view = 'entries';
```

Comment:
You shouldn't be setting `data.view` here, you should be setting this inside of `viewSwap`